### PR TITLE
Resolve merge conflicts for adding BME680

### PR DESCRIPTION
**Pull request by d-buchmann**
Sorry for bursting in...
Here's a mergable version of PR #253
I can't test this currently (at least not on real hw), but maybe it helps someone else at testing.

### CODE REVIEW

**Review by thijstriemstra — CHANGES_REQUESTED** [review]
can you also add the bme680 library to the dependencies in `setup.py`?

Also, remove any dead (e.g. commented) code.

thanks!

File: octoprint_enclosure/__init__.py [hunk 1]
```diff
@@ -978,45 +978,58 @@ def update_ui_inputs(self):
...
             if self.development_mode:
-                temp, hum = self.read_dummy_temp()
+                temp, hum, airquality = self.read_dummy_temp()
             else:
                 if sensor['temp_sensor_type'] in ["11", "22", "2302"]:
                     temp, hum = self.read_dht_temp(sensor['temp_sensor_type'], sensor['gpio_pin'])
                 elif sensor['temp_sensor_type'] == "18b20":
                     temp = self.read_18b20_temp(sensor['ds18b20_serial'])
                     hum = 0
+					airquality = 0
```
Comment (thijstriemstra) [inline octoprint_enclosure/__init__.py:988]: the indentation for `airquality` is incorrect in this file (it has an extra space)

File: octoprint_enclosure/BME680.py [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+sensor.select_gas_heater_profile(0)
+sensor.set_gas_status(bme680.ENABLE_GAS_MEAS)
+
+gas_reference = float(250000)
+hum_reference = float(40)
+getgasreference_count = int(0)
+
+
+def GetGasReference(gas_reference):
+  # Now run the sensor for a burn-in period, then use combination of relative humidity and gas resistance to estimate indoor air quality as a percentage.
```
Comment (thijstriemstra) [inline octoprint_enclosure/BME680.py:29]: the indentation for this file is not consistent and should be 4 spaces everywhere.

File: octoprint_enclosure/__init__.py (hunk 1 above)
Comment (d-buchmann) [reply octoprint_enclosure/__init__.py:988]: Thanks for the review. I fixed both indent errors, hope it's all correct (I'm not all too familiar with github + haven't even run the code yet)

Comment (thijstriemstra) [reply octoprint_enclosure/__init__.py:988]: no, i still see incorrect indentation

Comment (d-buchmann) [reply octoprint_enclosure/__init__.py]: Darn, those tabs :/

**Comment by thijstriemstra** [comment]
I think it also makes sense to use `sys.executable` like I did in the bme280 PR here: https://github.com/vitormhenrique/OctoPrint-Enclosure/pull/444/files#diff-a2404d7993f889e484a332d17e5f97235b6705017e48dfeefcc0161540f88272

I'll add a PR to fix this in all other code as well once this PR is merged.

**Comment by d-buchmann** [comment]
I'm not a python expert, but you sound like you know what you're doing so I basically just copy&pasted your code, if that's ok.

**Review by thijstriemstra — APPROVED** [review]
thanks! looks good to me.

File: octoprint_enclosure/BME680.py [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+    if (current_humidity < 38):
+        hum_score = float(0.25/hum_reference*current_humidity*100)
+    else:
+        hum_score = ((-0.25/(100-hum_reference)*current_humidity)+0.416666)*100
+
+#Calculate gas contribution to IAQ index
+gas_lower_limit = float(5000)   # Bad air quality limit
+gas_upper_limit = float(50000)  # Good air quality limit
+
+if (gas_reference > gas_upper_limit):
```
Comment (thijstriemstra) [inline octoprint_enclosure/BME680.py:73]: oh. these parentheses are not needed (same for line 75)

Comment (d-buchmann) [reply octoprint_enclosure/BME680.py]: Okay =)

**Pull request merged**